### ISSUE TITLE
Incorrect response on send SystemRequest (file name - /test)

### ISSUE THREAD

**Issue by DrachenkoAnastasiia** [smartdevicelink/sdl_core#955]
### Bug Report
Incorrect response on send SystemRequest (file name - /test)

##### Reproduction Steps
1. Send SystemRequest (choose any file from sdcard on mobile), specify request type - PROPRIETARY, Use file name - /test 

##### Expected Behavior
SDL responses with INVALID DATA result code

##### Observed Behavior
SDL responses with REJECTED result code

##### OS & Version Information
* OS/Version: Ubuntu 16.04
* SDL Core Version: a6c32b45832653d6b851be9b19355fa35f377e07
* Testing Against: ATF

##### Test Case, Sample Code, and / or Example App
[att_issue#955.zip](https://github.com/smartdevicelink/sdl_core/files/575465/att_issue.955.zip)

**Comment by LitvinenkoIra** [comment]
Priority Low.
Specific use-case related to used file name

**Comment by robinmk** [comment]
@LitvinenkoIra , why should the response be INVALID_DATA? 
Please add more details to the issue .

**Comment by KhrystynaDubovyk** [comment]
@robinmk, I guess this behaviour (and expected result) is based on the following requirements:

1. SDL must respond with 'INVALID_DATA, success:false' resultCode 
in case the name of file that the app requests to upload on the system 
contains `"/"` symbol (example: fileName: "../123.jpg")
2. SDL must respond with 'INVALID_DATA, success:false' resultCode 
in case the name of file that the app requests to upload on the system contains `"\"` (backslash) symbol (example: fileName: "..\123.jpg"

**Impacted RPC:**
-> PutFile (`<syncFileName>`)
-> SystemRequest (`<fileName>`)

**Comment by robinmk** [comment]
@KhrystynaDubovyk , Thank you for clarifying. Now I understand the problem.
Since the response is `REJECTED`, I agree that its a low priority bug.

**Comment by E-SAITO-TMC** [comment]
@theresalech -san This issue is a medium priority for Toyota. We tried it about 50 times and couldn't reproduce it. I would like to know if this issue occur in the latest version. And could you please include this issue in your plan for Q4 Core ?  Thank you.

**Comment by theresalech** [comment]
Hi @E-SAITO-TMC, we have documented that this is medium priority for Toyota, and will include for Steering Committee review and vote when planning the contents for the Q4 Core release, if issue is still reproducible.

@DrachenkoAnastasiia, can you please advise if you are seeing this issue when using Core 6.1?

**Comment by ychernysheva** [comment]
@theresalech This issue is not actual for Core 6.1. There is a need in actualization of corresponding test script, after that it can be delivered on Livio Team review.

**Comment by theresalech** [comment]
Hi @ychernysheva, please let us know once the corresponding test script is ready for review.  We can then review the test script and close this Core issue as not reproducible.

**Comment by dboltovskyi** [comment]
@theresalech Please notice PR with script is ready for PM review: https://github.com/smartdevicelink/sdl_atf_test_scripts/pull/2071

**Comment by ShobhitAd** [comment]
Issue not reproducible in latest develop(https://github.com/smartdevicelink/sdl_core/commit/ef2b88520b4224390ffd332f737fdf5573def60e).
